gym_compete/policy.py

```diff
@@ -212,19 +213,20 @@ def loop_fn(time, cell_output, cell_state, loop_state):
                     last_out = seq_to_batch(last_out)
                     self.state_out.append(final_state)
 
-                    return last_out
+                    return last_out, input_seq
```

Comment:
Is `input_seq` really what we want? It's the output of the last hidden layer before going into the LSTM. If we're going to pick just one layer it seems like a reasonable one to pick, but why not expose all the layers so we can choose which one to look at during experiments? This would also be needed to support some more advanced methods like the HER idea where we'd start at the first layer and work up to deeper layers over time.